### DISCUSSION TITLE
Seeking Feedback on My Google Summer of Code Proposal: anomalibGPT

### DISCUSSION THREAD

**Discussion by Bepitic** [open-edge-platform/anomalib#1932] [General]
Hello, anomalib Community,

I'm Bepitic, I've lurking around for a couple of months and am excited to connect! As a quick intro, I'm passionate about ML and currently drafting a proposal for Google Summer of Code focused on anomalibGPT, using VLM on anomalib with openAI.

I'd greatly appreciate your insights on my proposal, which can be found here: [Bepitic-AnomalibGPT.pdf](https://github.com/openvinotoolkit/anomalib/files/14779924/Bepitic-AnomalibGPT.pdf). Any feedback to help refine and improve it would be wonderful.
@samet-akcay @djdameln @ashwinvaidya17 
Thank you so much for your time and thoughts!

**Comment by monkeycc** [comment]
https://github.com/openvinotoolkit/anomalib/discussions/1351

**Reply by Bepitic** [reply]
Thank you, now I have more certainty that it may work :)

**Comment by samet-akcay** [comment]
Closing since the application period is now over.
